### PR TITLE
Stable/5.0 beta AUTO-8 Openstack cli execution fixes for MOSK

### PR DESCRIPTION
**Pull request by savitha-p**
Trial sanity run report: [192.168.1.62:8080/job/TVO-Automation/job/MOSK/job/MOSK-1_Sanity/10/HTML_20Report/](http://192.168.1.62:8080/job/TVO-Automation/job/MOSK/job/MOSK-1_Sanity/10/HTML_20Report/)

Trial functional run report: [192.168.1.62:8080/job/TVO-Automation/job/MOSK/job/MOSK-1_Func/3/HTML_20Reports/](http://192.168.1.62:8080/job/TVO-Automation/job/MOSK/job/MOSK-1_Func/3/HTML_20Reports/)

### CODE REVIEW

File: fetch_resources.sh [hunk 1]
```diff
@@ -28,7 +28,11 @@ sed -i "/PYTHON_CMD=/c PYTHON_CMD=\"$TEMPEST_VENV_DIR/bin/python$PYTHON_VERSION\
 
 if [[ "$AUTH_URL" =~ "https" ]]
 then
-    OPENSTACK_CMD="openstack --insecure"
+    cd /root
```
Comment (nasharatm) [inline fetch_resources.sh]: Do we need provision for "--insecure" flag too?

Comment (savitha-p) [reply fetch_resources.sh]: That is automatically handled in keystone pod.

**Review by nasharatm — APPROVED** [review]

**Review by priyankagupte15 — APPROVED** [review]

File: fetch_resources.sh [hunk 2]
```diff
@@ -26,12 +26,15 @@ sed -i "2i export PYTHON_VERSION=$PYTHON_VERSION" run_tempest.sh
 sed -i "/PYTHON_CMD=/c PYTHON_CMD=\"$TEMPEST_VENV_DIR/bin/python$PYTHON_VERSION\"" sanity-run.sh
 sed -i "/PYTHON_CMD=/c PYTHON_CMD=\"$TEMPEST_VENV_DIR/bin/python$PYTHON_VERSION\"" master-run.sh
 
+cd /root
+eval "$(<env.sh)"
+export cli_pod=$(kubectl -n openstack get pod -l application=keystone,component=client -ojsonpath='{.items[*].metadata.name}')
```
Comment (SuhasTA4) [inline fetch_resources.sh:31]: Please confirm parameter here. You gave "-ojsonpath". I think you missed to give space between  "-o" and "jsonpath".

**Review by SuhasTA4 — CHANGES_REQUESTED** [review]

File: fetch_resources.sh (hunk 2 above)
Comment (savitha-p) [reply fetch_resources.sh:31]: It works both ways, with space and without space

**Review by SuhasTA4 — APPROVED** [review]

**Pull request merged**